src/Groups/directproducts.jl

```diff
@@ -168,16 +170,16 @@ julia> K = symmetric_group(2)
 Sym(2)
 
 julia> G = direct_product(H, K)
-Direct product of
- Sym(3)
- Sym(2)
+Sym(3) x Sym(2)
```

Comment:
Here I am also torn: perhaps the original output should be kept, and `Sym(3) x Sym(2)` just used for oneline resp. terse printing?